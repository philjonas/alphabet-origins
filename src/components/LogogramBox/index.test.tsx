import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() })
import { LogogramBox } from '.';


test('matches snapshot', () => {
  const wrapper = shallow(<LogogramBox />)
  expect(toJson(wrapper)).toMatchSnapshot();
})