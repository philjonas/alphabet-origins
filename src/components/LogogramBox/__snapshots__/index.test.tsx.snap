// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="container-logogram"
>
  <div
    className="symbol"
  />
  <div
    className="legend"
  />
</div>
`;
